model/Storage.inc.php

```diff
@@ -236,7 +237,7 @@ public static function queueUpload($userID, Zotero_StorageFileInfo $info) {
 				$info->charset
 			)
 		);
-		
+
```

Comment:
Looks like some of the empty lines on dataserver are indented and some aren't, even they should. My editor is fighting with them :)